esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -46,6 +46,8 @@
 #define FILE_DESPRAY_URL    "FileDesprayAccess"
 #define WUDETAILS_REFRESH_MINS 1
 
+const unsigned dropZoneFileSearchMaxFiles = 1000;
```

Comment:
Question: Would this be better if it could be specified in the request?  Are there situations where 1000 is far too many, or too few?

Reply:
A user may not know (or do not care) this resource limitation. The user may always specify a big (or huge) number.

Reply:
If we are now returning a flat list (based on this PR) and there is no longer any recursion and linear search and we also do not allow an emply/blank or '*' filter (based on this PR) then perhaps we do not need a limit or it could be larger than 1000.

Reply:
For performance and resource safe, I think we do need a limit.